### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bytebebop**

### CODE REVIEW

**Review by oscar-castaneda-dev — COMMENTED** [review]
something

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # example
+
+this is an example
```
Comment (oscar-castaneda-dev) [inline README.md:3]: something

Comment (bytebebop) [reply README.md:3]: yes

Comment (oscar-castaneda-dev) [reply README.md:3]: yes

**Pull request merged**